client/src/app/+about/about-follows/about-follows.component.html

```diff
@@ -1,22 +1,26 @@
-<div class="row" myInfiniteScroller [autoInit]="true" (nearOfBottom)="onNearOfBottom()" [dataObservable]="onDataSubject.asObservable()">
+<div class="row">
   <div class="col-xl-6 col-md-12">
     <div i18n class="subtitle">Followers instances</div>
 
     <div i18n class="no-results" *ngIf="followersPagination.totalItems === 0">This instance does not have instances followers.</div>
-
+    
     <a *ngFor="let follower of followers" [href]="buildLink(follower)" target="_blank" rel="noopener noreferrer">
       {{ follower }}
     </a>
+
+    <a id="showMore" href="javascript:;" *ngIf="!showMoreFollowers" (click)="loadAllFollowers()" (click)= "showMoreFollowers=true">Show full list</a> 
   </div>
 
   <div class="col-xl-6 col-md-12">
     <div i18n class="subtitle">Followings instances</div>
 
     <div i18n class="no-results" *ngIf="followingsPagination.totalItems === 0">This instance does not have instances followings.</div>
-
+    
     <a *ngFor="let following of followings" [href]="buildLink(following)" target="_blank" rel="noopener noreferrer">
       {{ following }}
     </a>
+    
+    <a id="showMore" href="javascript:;" *ngIf="!showMoreFollowings" (click)="loadAllFollowings()" (click)= "showMoreFollowings=true">Show full list</a>   
```

Comment:
why do you use `href="javascript:;"`?

Reply:
please use a class instead of an id ; they benefit from the component encapsulation so ids are not as useful.

Reply:
please use the `i18n` attribute.

Reply:
please remove trailing spaces.